### DISCUSSION TITLE
Tools to add

### DISCUSSION THREAD

**Discussion by 0xtavian** [pry0cc/axiom#393] [General]
suggest a tool by posting a link to the github
the only absolute requirement is that its open source

https://github.com/Cerbrutus-BruteForcer/cerbrutus 

https://github.com/cyberark/BlobHunter
https://github.com/detectify/page-fetch

**Comment by ResistanceIsUseless** [comment]
https://github.com/neex/http2smugl
https://github.com/blacklanternsecurity/MANSPIDER
https://github.com/dwisiswant0/crlfuzz
https://github.com/xm1k3/cent


Maybe we make reconFTW image into an BB automation image with docker images where possible?
https://github.com/Tib3rius/AutoRecon
https://github.com/reconmap/reconmap
https://github.com/intrigueio/intrigue-core
https://github.com/KathanP19/JSFScan.sh
https://github.com/Chudry/Xerror
https://github.com/reconness/reconness

**Reply by ResistanceIsUseless** [reply]
looks like https://github.com/neex/http2smugl is already there. so ignore that.

**Comment by ghost** [comment]
Could you consider adding dirsearch?

**Comment by EduardTalianu** [comment]
https://github.com/0xZDH/o365spray
https://github.com/blacklanternsecurity/TREVORspray
https://gitlab.com/shodan-public/nrich
https://github.com/s0md3v/Smap

**Comment by georgebecerescu** [comment]
sliver
